portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,33 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random movie to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomMovie() {
+  const movies =
+      ['The Greatest Showman', 'Kaguya-hime no Monogatari', 'The Imitation Game', 'The Man Who Knew Infinity', 'Spiderman into the spider verse', 'Mary Shelley'];
```

Comment:
As you might have guessed by now... I'm going to recommend the [Google JavaScript style guide](https://google.github.io/styleguide/jsguide.html) to format your JavaScript code.